dapr/proto/runtime/v1/dapr.proto

```diff
@@ -1295,6 +1295,9 @@ message ConversationAlpha1Request {
 
   // The metadata passing to conversation components.
   map<string, string> metadata = 5;
+
```

Comment:
Nit: do we want to rename `conversationContext` to `context` since `conversation` is superfluous as it's derived through context (?)

The description is also confusing, and suggests we should use a new field name altogether- such as `session`, `sessionID`,  `chatID` etc.

```
the Id of an existing chat room (like in ChatGPT)
```

Reply:
Changed, including description